server/dash/dashboards/neighborhood_recent.py

```diff
@@ -186,7 +171,12 @@ def update_text(selected_council):
 def update_figure(selected_council):
     figure_df = df.query(f"councilName == '{selected_council}' and createdDate >= '{start_date}'").groupby(['createdDate', 'typeName'])['srnumber'].count().reset_index()  # noqa
     figure_df.typeName = figure_df.typeName.map(lambda x: '<br>'.join(textwrap.wrap(x, width=16)))  # noqa
-
+    if figure_df.shape[0] == 0:
```

Comment:
Ditto, please add comment explaining why we need to check this

Reply:
Added. Thanks!